CHANGELOG.md

```diff
@@ -1,5 +1,44 @@
 # osquery Changelog
 
+
+<a name="5.2.1"></a>
+## [5.2.1](https://github.com/osquery/osquery/releases/tag/5.2.1)
+
+[Git Commits](https://github.com/osquery/osquery/compare/5.1.0...5.2.1)
+
+Osquery 5.2.1 brings native Apple Silicon (M1) support to the macOS
+platform. It also represents a comprehensive review and update of our
+third-party dependancies. To support this work, the developer docs
```

Comment:
`dependencies`

Ok, I'm just going to list out the libraries that were updated, and to which version we updated them, if that information was easy for me to find. I think this can go in the changelog, as it will make it easier for the people monitoring these types of things in Synopsis BlackDuck and stuff.

### Third-party Libraries

- Update augeas to version 1.12.0 and update its transient dependency osquery/gnulib (#7330)
- Update aws-sdk to version 1.9.177 (#7330)
- Update boost to version 1.78.0 (#7330)
- Update googletest to version 1.11.0 (#7330)
- Update glog to version 0.5.0 (#7330)
- Update libarchive to version 3.5.2 (#7330)
- Update libcap (#7330)
- Update libmagic/file to version 5.40 (#7330)
- Update librdkafka to version 1.6.2 (#7330)
- Update libxml2 to version 2.9.12 (#7330)
- Update linenoise-ng (#7330)
- Update lzma/xz to version 5.2.2 (#7330)
- Update OpenSSL to version 1.1.1l (#7330)
- Update rocksdb to 6.27.3 (#7330)
- Update sleuthkit to 4.11.1 (#7330)
- Update sqlite to version 3.37.0 (#7426) (#7330)
- Update ssdeep-cpp to 2.14.1 (#7330)
- Update thrift to 0.15.0 (#7330)
- Update yara to version 4.1.3 (#7330)
- Update zlib to version 1.2.11 (#7330)
- Update zstd to version 1.5.1 (#7330)

Reply:
Some corrections/additions:

zlib has not been updated in version
zstd version is 1.4.0, not 1.5.1
gflags is missing here, and it has been updated to version 2.2.2
libcap has been updated to version 1.2.59
aws-sdk version is 1.9.116 not 1.9.177
boost version is 1.77 not 1.78
librdkafka version is 1.8.0 not 1.6.2
rocksdb version is 6.22.1 not 6.27.3
sleuthkit version is 4.11.0 not 4.11.1
ssdeep-cpp was already at 2.14.1 and now is at its latest commit https://github.com/ssdeep-project/ssdeep/commit/d8705da60369a27a2b5d22da488416e8ab348d8f

Reply:
This is a lot easier when there are different PRs for each.

Reply:
Okay, I think I added them all in